pkg/common/option_test.go

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+import (
+	"regexp"
+	"testing"
+)
+
+func TestUint8(t *testing.T) {
+	var u8 uint8
+	u := NewUint8Value(128, &u8)
+	if u8 != 128 {
+		t.Errorf("expected %d, result %d", 128, u8)
+	}
+
+	str := "100"
+	u8 = 100
```

Comment:
```suggestion
	u8 = 200
```

Reply:
Those unit tests could confuse people, I am going to rewrite them.

Reply:
I propose to remove `option.go`, it is not used in our code anymore.
Also propose to remove `go.go` (tricky and never used) and `time.go` (tricky, only referenced in an unused logger function, can be replaced by standard library).

Reply:
Agree with removing `option.go` in `pkg/common`, since we had changed to use `pkg/option` already. 
The same to `go.go`.

Reply:
I suggest another dedicated PR to clean up the source code.  this one is just only for the unit tests.